docs/authentication/saml-at-clerk.mdx

```diff
@@ -134,7 +134,7 @@ Everything should be set up now for you to try out authentication via SAML. Go t
 
 **I’ve enabled other strategies but they don’t work.**
 
-It is expected that once Enterprise SSO is enabled in an application, there should be no other authentication methods applicable. This is in line with an organization’s intent to manage their users’ identity from one place.
+A Clerk application can have multiple authentication strategies, but a domain that enables Enterprise SSO can not. Once Enterprise SSO is enabled for an domain, there can be no other authentication methods for that domain. This is in line with an organization’s intent to manage their users’ identity from one place. This will allow your Clerk application to enable Enterprise SSO connections for certain domains while others use non-Enterprise SSO methods depending on each organization's needs.
```

Comment:
```suggestion
A Clerk application can have multiple authentication strategies, but a domain that enables Enterprise SSO can not. Once Enterprise SSO is enabled for a domain, there can be no other authentication methods for that specific domain. This is in line with an organization’s intent to manage their users’ identity from one place. This will allow your Clerk application to enable Enterprise SSO connections for certain domains while others use non-Enterprise SSO methods depending on each organization's needs.
```